plugins/defaultplugins/common/model/l3/l3.proto

```diff
@@ -33,26 +33,34 @@ message ArpTable {
 
 /* Proxy ARP ranges */
 message ProxyArpRanges {
-    message ProxyArpRange {
-        string range_ip_start = 1;                  /* low (starting) IP address of the proxy ARP range */
-        string range_ip_end = 2;                    /* high (last) IP address of the proxy ARP range  */
+    message RangeList {
+        string name = 1;                                /* unique proxy ARP range config identifier */
+        message Ranges {
+            string first_ip = 1;                        /* first (starting) IP address of the proxy ARP range */
+            string last_ip = 2;                         /* last (ending) IP address of the proxy ARP range  */
+        }
+        repeated Ranges ranges = 2;
     }
-    repeated ProxyArpRange proxy_arp_ranges = 100;  /* list of proxy arp ranges */
+    repeated RangeList ranges = 100;                    /* list of proxy arp ranges */
 }
 
 /* Proxy ARP interfaces */
 message ProxyArpInterfaces {
-    message ProxyArpInterface {
-        string interface = 1;                               /* interface name */
+    message InterfaceList {
+        string name = 1;                                /* unique proxy ARP interface config conigurator identifier */
+        message Interfaces {
+            string interface = 1;                       /* name of the interface proxy arp is enabled for */
```

Comment:
This should be renamed to `name` so it's consistent with other similar usages elsewhere.